includes/class-wc-ajax.php

```diff
@@ -1300,6 +1299,18 @@ public static function remove_order_item() {
 			$order->calculate_taxes( $calculate_tax_args );
 			$order->calculate_totals( false );
 
+			/**
+			 * Fires after order items are removed.
+			 *
+			 * @since 4.9.0
```

Comment:
```suggestion
			 * @since 5.1.0
```

Reply:
@claudiosanches - Done :) If in the future I should not leave a msg, but instead should use "Resolve conversation" just let me know.

Reply:
@claudiosanches - Pardon my n00b-iness but can you tell me what I need to (un) do about this?

https://github.com/woocommerce/woocommerce/actions/runs/528661392